_posts/2021-09-13-changelog-155.markdown

```diff
@@ -0,0 +1,73 @@
+
+* [#7772] Show warning/errors during fetching stdlib info in [Sync View](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-cargo-tool-window.html)
+
+* [#7777] Enable by default code completion ranking for Rust based on machine learning. Now the completion suggestions are sorted by their relevance based on the rules learned from data we have gathered anonymously during our EAPs. Note that we have not collected any source code, only information about your interactions with the code completion UI. In case of any issues, you can disable the new ranking model via `Editor | General | Code Completion | Machine Learning-Assisted Completion` settings
+
+* [#7769], [#7751] Correctly parse `unsafe` keyword in extern and module blocks, annotate them as errors
+  <img src="/assets/posts/changelog-155/unsafe-extern-mod.png" alt="Parse and annotate unsafe in extern and module blocks" width="700px">
+
+* [#7734] Introduce new icons for macro items, providing `macro_rules!` and `macros 2.0` items with distinctive ones
+  <img src="/assets/posts/changelog-155/new-macro-icons.png" alt="New macro items icons" width="700px">
+
+* [#7663] Support `Move Element Left/Right` action for predicates in `where` clause
+  {% include gif-img.html path="/assets/posts/changelog-155/where-move" w="700px" %}
+
+* [#7757] Parse and annotate `const_generics_defaults` compiler feature
+
+* [#3790] Deprecated item inspection now correctly handles items that will be deprecated in a future version (by [@mibac138])
+
+## Fixes
+
+* [#7795] Improve parsing of const generic arguments for `const_generics_defaults` compiler feature, provide inspection and quick-fix for expressions that must be enclosed
+
+* [#7790] Generate [const generics](https://rust-lang.github.io/rfcs/2000-const-generics.html) by `Implement Members` quick-fix properly
+
+* [#7758] Don't fetch stdlib metadata for `rustc` project since it's already fetched during project structure setup
+
+* [#7752] Properly render negative `impl` items in `Go to implementation` popup
+
+* [#7728] Fix DTrace profiler integration with CLion 2021.2
+
+* [#7726] Correctly print restricted visibility of items in quick documentation instead of always printing "pub" (by [@afetisov])
+
+* [#7637] Use existing impl block if possible in `Extract function` refactoring (<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>M</kbd>)
+
+* [#7506] Fix macros and generated items support for WSL toolchains
+
+## Internal Improvements
+
+* [#7740] Drop CLA requirement. There is no more need to sign it to make a contribution into plugin
```

Comment:
Do we require to add a username into `CONTRIBUTORS.txt` file anymore? Should we mention it here?

Reply:
No, we dropped this mention as well but I think it's not worth mentioning